internal/staticlint/staticlint.go

```diff
@@ -13,6 +13,20 @@ var ExitFromMainAnalyzer = &analysis.Analyzer{
 	Run:  run,
 }
 
+func ispectFunc(decl ast.Decl) {
+	ast.Inspect(decl, func(n ast.Node) bool {
+		// только вызовы функций
+		if c, ok := n.(*ast.CallExpr); ok {
```

Comment:
Я имел в виду, что-то типа:

```suggestion
		c, ok := n.(*ast.CallExpr)
		if !ok {
```